src/pept-compare/methods.py

```diff
@@ -18,10 +19,13 @@
 grey = "#ebf5ee"
 
 
-def read_files():
+def read_files_gui():
     root = tk.Tk()
     root.withdraw()
     filenames = askopenfilenames(initialdir="/Documents/GitHub/kand/example_files", title="Open files", multiple=True, )
+    return make_peptide_dfs(filenames)
+
+def make_peptide_dfs(filenames : List):
```

Comment:
It makes sense having these as two different methods. That way files can be read by automated tests or a command-line interface, etc.